packages/devtools/lib/src/vm_service_wrapper.dart

```diff
@@ -142,6 +142,19 @@ class VmServiceWrapper implements VmService {
     return _trackFuture(_vmService.getCpuProfile(isolateId, tags));
   }
 
+  Future<Response> getCpuProfileTimeline(
+      String isolateId, int origin, int extent) async {
+    return _trackFuture(callMethod(
```

Comment:
add a TODO to keep track of a list of all private methods we are currently using to share with the VM team to make them public.

Reply:
done.